File: Ejercicios/5. Duck/Net/Before/Duck/DecoyDuck.cs
namespace Duck
{
    public class DecoyDuck : Duck
    {
        public override object Fly()
        {
            return DuckMessages.CantFly;
        }

        public override object Quack()
        {
            return DuckMessages.Silence;
        }

        public override object Display()
        {
            return DuckMessages.ModelDisplay;
        }
    }
}

File: Ejercicios/5. Duck/Java/Before/src/main/duck/DecoyDuck.java
package duck;

public class DecoyDuck  extends Duck{
    public Object fly()
    {
        return DuckMessages.CantFly;
    }

    public Object quack()
    {
        return DuckMessages.Silence;
    }

    public Object display()
    {
        return DuckMessages.ModelDisplay;
    }
}
